runtime/auth/aws-signing-common/common/src/aws/smithy/kotlin/runtime/auth/awssigning/AwsSigningConfig.kt

```diff
@@ -172,6 +172,11 @@ public class AwsSigningConfig(builder: Builder) {
      */
     public val logRequest: Boolean = builder.logRequest
 
+    /**
+     * Determines the checksum to add to the canonical request query parameters before signing.
```

Comment:
docs: KDocs should be more descriptive about the contents of the `Pair`, what is first, second?